### ISSUE TITLE
A issue in pygaps.modelIsotherms

### ISSUE THREAD

**Issue by libinnever** [pauliacomi/pyGAPS#42]
Dear Dr Iacomi,
I am a PhD student working on gas separation, and recently I want to use your pygaps for some IAST calculations. When I tried to use pre-defined parameters for a model following (https://pygaps.readthedocs.io/en/master/examples/modelling.html,part [11]), I found that there is no output plot. After adding some print() to the code, I found out that it seems pgm.get_isotherm_model can not get params from params class. The following pictures are what I've changed and the result. You can see that even though I changed the "K" to 40 and 'pressure_range ' to (0.01,5), the resulted params is still 20 for K (0.01, 2) for pressure range(First line of the last picture, generated due to the print() in the sencond picture), and the parameters for the model are still nan(line 2-9 and line 11, generated due to the print() in the first picture). Could you help me check it and let me know what I should do to make it work?
p.s. I am using the spyder in anaconda to run the python(3.8). The system of my laptop is windows 10.
![thumbnail_image](https://user-images.githubusercontent.com/87354846/216507502-4c75d513-cb71-49ab-942d-0578b6456584.png)
![thumbnail_image](https://user-images.githubusercontent.com/87354846/216507548-b719740f-1760-43e9-bb83-db929bde45b4.png)
![thumbnail_image](https://user-images.githubusercontent.com/87354846/216507558-cfbddd5d-1626-438b-ac47-2dc7293f29d6.png)

Many thanks,

Libin

**Comment by pauliacomi** [comment]
Dear Libin,

I realised that this was an example which relied on an older version of the code.
To make it work you can do the following, the difference is subtle:

```python
model = pgm.get_isotherm_model(
    'Langmuir',
    parameters={
        "K": 20,
        "n_m": 2
    },
    pressure_range=(0.01, 2),
)
model_iso = pg.ModelIsotherm(
    material='carbon',
    adsorbate='N2',
    temperature=77,
    model=model,
)
model_iso.plot()

```

Also I believe very recently jupyter changed its default matplotlib behaviour. You can add 
```python
%matplotlib inline
```
at the top to make python clear graphs after every cell.
Let me know if it works for you.

I will correct the example in the next few days. Thank you for bringing it to my attention.

Best,
Paul

**Comment by libinnever** [comment]
Hi Paul,
It works fine following your instructions. 

Many thanks,
Libin

**Comment by pauliacomi** [comment]
Fixed in [d2e6275](https://github.com/pauliacomi/pyGAPS/commit/d2e62753643695b230fddea6cc7f82c47203cfc3)

Closing this now, let me know if there's anything else I can help with.

Best,
Paul
